### DISCUSSION TITLE
Mix 2 units of same type in 1 graph

### DISCUSSION THREAD

**Discussion by JanSchnacki** [kalkih/mini-graph-card#735] [Ideas]
Hi,

I have an interesting situation. I have 2 entities that report wind speed, but one of them produces data with the unit "m/s" and one of them produces data with "km/h". I wanted to combine both in one graph, since they are directly related. While the unit is displayed correctly, the graph does not scale them so they reflect the correct relative height. 

While I expected that, I thought it might be possible to do this manually. Unfortunately the "value_factor" option apparently only exists on the card level, not on the entity level. 

Is this feasible?

**Comment by ildar170975** [comment]
Duplicate of https://github.com/kalkih/mini-graph-card/issues/591

**Reply by JanSchnacki** [reply]
That's it. Thanks for pointing me to that ticket. I admit I didn't even think about looking at enhancement tickets first. I'm sorry for that.
